dev/cognitiveservices/data-plane/Language/questionanswering-authoring.json

```diff
@@ -27,7 +41,7 @@
     ]
   },
   "paths": {
-    "/query-knowledgebases/projects": {
+    "/authoring/query-knowledgebases/projects": {
```

Comment:
I'd just set the base location in the top of the swagger to avoid repeating this. Either way, though, it's a breaking change. If you GA'd the QA authoring APIs, you cannot make this change now without an official deprecation notice and supporting the old way for 3 years, not to mention we'd have to do some version conditions in the SDKs. See https://aka.ms/AzBreakingChangesPolicy for details.

Reply:
Yes, this already GA'd: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/data-plane/Language/stable/2021-10-01/questionanswering-authoring.json

This is a breaking change and will have to be reviewed. /cc @JeffreyRichter @johanste

Reply:
@heaths To avoid the breaking change I have added new paths with "/authoring/" and the old paths are still as such.

Reply:
Are these just copies now? That will end up breaking codegen. Instead, just replace the old ones or, as I said, set the base path once at the top and the whole `paths` section will be unchanged for a nice, clean diff.